app/src/main/java/net/gsantner/opoc/ui/SearchOrCustomTextDialog.java

```diff
@@ -342,10 +342,16 @@ public static void showMultiChoiceDialogWithSearchFilterUI(final Activity activi
         };
 
         // Helper function to append selection count to OK button
+        final String okString = dopt.okButtonText != 0 ? activity.getString(dopt.okButtonText) : "";
```

Comment:
Unrelated fix for when okButtonText == 0.

Text actions calls are surrounded with a try-catch all, so exceptions caused here were not caught